src/foam/core/reflow/Flow.js

```diff
@@ -143,6 +143,7 @@ foam.CLASS({
       class: 'Int',
       name: 'revision',
       transient: true,
+      value: 1,
```

Comment:
Why this change? I think it needs to start at 0 to be compatible with the MementoMgr.position.

Reply:
Save validates on it - https://github.com/kgrgreer/foam3/pull/3990/files#diff-7f96a09d61a35220c0fe8ea9b9c2e94d6a16681c59a755aef2c2fc9ba9ae1cb9L176

from a user perspective that isn't clear. So either we remove it from the enabled function or increment it.